scripts/ci_test_mqtt.sh

```diff
@@ -35,9 +35,11 @@ if [ "$TRAVIS_OS_NAME" = "linux" ]; then
     ./output/bin/iot_demo_mqtt $DEMO_OPTIONS
```

Comment:
Demos don't need to run for coverage job.

Reply:
Don't want to add more special logic to avoid running demos

Reply:
It's a waste of time to run the demos exactly because the data is stripped out later.  Why take the time to run the demos if the data will be ignored?